src/Controller/Component/RequestHandlerComponent.php

```diff
@@ -300,6 +300,7 @@ public function beforeRedirect(Event $event, $url, Response $response)
                 'REQUEST_METHOD' => 'GET'
             ]
         ]));
+        $response->statusCode(200);
         $response->send();
         $response->stop();
     }
```

Comment:
It does not have to be handled in the controller, returning the response here will bubble up to the dispatcher